deploy/charts/harvester/templates/cattle-system.yaml

```diff
@@ -15,10 +15,66 @@ spec:
   secrets:
     - name: harvester-additional-ca
       path: /ca
-  version: v1.0.2
+  version: v1.1.0
   upgrade:
     image: {{ .Values.generalJob.image.repository }}:{{ .Values.generalJob.image.tag }}
     command: ["/bin/sh", "-c"]
     args:
-      - cp /ca/additional-ca.pem /host/etc/pki/trust/anchors;
-      - chroot /host update-ca-certificates
+      - echo "Update additional ca ...";
+        cp /ca/additional-ca.pem /host/etc/pki/trust/anchors;
+        chroot /host update-ca-certificates;
+        echo "Done";
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: harvester-update-containerd-registry-script
+  namespace: cattle-system
+type: Opaque
+stringData:
+  update_registries.sh: |
+    #!/bin/sh
+    RESTART=true
+    if [ -f "/etc/rancher/rke2/registries.yaml" ]; then
+      if [ cmp -s "/run/system-upgrade/secrets/registries/registries.yaml" "/etc/rancher/rke2/registries.yaml" ]; then
+        RESTART=false
+        echo "/etc/rancher/rke2/registries.yaml is not changed"
+      fi
+    else
+      echo "File /etc/rancher/rke2/registries.yaml doesn't exist"
+      if [ ! -s "/run/system-upgrade/secrets/registries/registries.yaml" ]; then
+        RESTART=false
+        echo "New registries.yaml is empty"
+      fi
+    fi
+    if [ "$RESTART" == true ]; then
+      echo "Update /etc/rancher/rke2 ..."
+      cp /run/system-upgrade/secrets/registries/registries.yaml /etc/rancher/rke2
+      echo "Restart RKE2 ..."
+      kill $(pgrep rke2)
```

Comment:
@w13915984028 agree we merged for rc1 first and @FrankYang0529 can answer comments the next day. Thanks.

Reply:
@bk201 @FrankYang0529 

Good to merge now, those minor questions could be checked later. thanks.

Reply:
I tried `systemctl` before, but it can't work.

ref: https://stackoverflow.com/questions/49285658/how-to-solve-docker-issue-failed-to-connect-to-bus-no-such-file-or-directory